### ISSUE TITLE
Misaligned text on status bar when Linter is enabled

### ISSUE THREAD

**Issue by ZoneSix** [jesseweed/seti-ui#471]
The text on the left side in the statusbar is misaligned when using Linter.
Atom version: 1.23.3
Seti-UI version: 1.10.0
![atom-seti-ui-bottom-fault](https://user-images.githubusercontent.com/8831008/35961269-5841f84c-0cad-11e8-9427-4da6d8cf5ce0.png)

**Comment by jesseweed** [comment]
Weird I use linter and haven't seen that issue, but I'll look into it.

**Comment by warsier** [comment]
I have observed this misaligned issue on Windows, while on Linux systems the status bar works correctly.

**Comment by mccallum-sgd** [comment]
Same here on Windows 10 Home (version: 1709, build: 16299.309).
Atom version: 1.25.0
Seti-UI version: 1.10.0

Note that this happens regardless of compact mode, and whether the windows is maximized or not (I also tried resizing it).

**Comment by Xenospark** [comment]
I have only observed this behavior with 'Compact Mode' enabled in seti-ui Settings.
I have only been able to confirm this behavior on Windows 7/10.
I Cannot reproduce this behavior on any Linux distro that I have access to.

So far I have only come up with a workaround proposal for windows 7/10 users experiencing this issue.

Dig into C:\Users\\{your-user-name}\\.atom\packages\seti-ui\styles\components\status-bar.less

Goto line 21 you will find:
  ```
.status-bar-left {
      padding: 0;
    }
```

Change it to: 
 ```
 .status-bar-left {
      padding: 0;
      display: inline-flex;
    }
```

be advised - any future updates applied to seti-ui is likely to over-write your changes.
